vic/drivers/image/src/get_global_domain.c

```diff
@@ -103,15 +100,15 @@ get_global_domain(char          *nc_name,
     }
 
     // Get number of lat/lon dimensions.
-    options.COORD_DIMS_OUT = get_nc_varndimensions(nc_name,
-                                                   options.DOMAIN_LON_VAR);
-    if (options.COORD_DIMS_OUT !=
-        get_nc_varndimensions(nc_name, options.DOMAIN_LAT_VAR)) {
+    global_domain->info.n_coord_dims = get_nc_varndimensions(nc_name,
+                                                   global_domain->info.lon_var);
+    if (global_domain->info.n_coord_dims !=
+        (size_t) get_nc_varndimensions(nc_name, global_domain->info.lat_var)) {
         log_err("Un even number of dimensions for %s and %s in: %s",
-                options.DOMAIN_LON_VAR, options.DOMAIN_LAT_VAR, nc_name);
+                global_domain->info.lon_var, global_domain->info.lat_var, nc_name);
     }
 
-    if (options.COORD_DIMS_OUT == 1) {
+    if (global_domain->info.n_coord_dims == 1) {
         double *varLon = NULL;
```

Comment:
in VIC we declare all the variables at the top of the function. Even though that is no longer necessary in newer C versions, it would be nice to stick to that. So please gather variable declarations at the top.